modules/nw-installing-external-dns-operator.adoc

```diff
@@ -13,7 +13,6 @@ You can install the External DNS Operator by using the {product-title} OperatorH
 . Click *Operators* -> *OperatorHub* in the {product-title} web console.
 . Click *External DNS Operator*.
   You can use the *Filter by keyword* text box or the filter list to search for External DNS Operator from the list of Operators.
-. Select the `external-dns-operator` namespace.
```

Comment:
Do not remove this step.

Reply:
Agree if [the proposed detail](https://github.com/openshift/openshift-docs/pull/76161/files#r1622542453) will be added.